ui/app/src/modules/Content/Publish/PublishDialog.tsx

```diff
@@ -82,65 +89,94 @@ export const selectAllDeps = (setChecked: Function, items: Item[]) => {
 
 export const paths = (checked: any) => (
   Object.entries({ ...checked })
-    .filter(([key, value]) => value === true)
-    .map(([key]) => key)
+  .filter(([key, value]) => value === true)
```

Comment:
Did you import the code style settings? Missing indentation.

Reply:
Yes it is updated, that's weird